setup-earthly/libs/setup-earthly.js

```diff
@@ -0,0 +1,36 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
+
+const IS_WINDOWS = process.platform === "win32";
+const IS_MAC = process.platform === "darwin";
+
+try {
+  console.log(`Setting up earthly`);
+
+  install_cmd = "sudo /bin/sh -c 'wget https://github.com/earthly/earthly/releases/latest/download/earthly-linux-amd64 -O /usr/local/bin/earthly && chmod +x /usr/local/bin/earthly'"
```

Comment:
I wouldn't use `latest` in CI. Perhaps the user can specify an Earthly version

Reply:
I'll allow users to specify it as:

```
      - uses: earthly/actions/setup-earthly@df239341c368eb73f32ef4c429aea5f0b61a280a
        with:
          earthly-version: '0.5.2'

```

and have it default to latest if omitted.